components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -211,86 +177,141 @@ var img = "";
+			if (!$('#stepsConfRow').is(":visible")) {
+				$(jQuery('#stepsConfRow')).toggle();
+			}
+			if (stepOrder == 1) {
+				$('#subject_step_' + stepOrder).attr('checked', true);
+				$('#attribute_step_' + stepOrder).attr('checked', true);
+			}
+		})
+		//TODO Add help text and document link
+		var template = {
+			"templateList": [{
+				"category": "Request Based",
+				"type": [{
+					"name": "ACR",
+					"img": "./images/user.gif",
+					"code": "function(context) {\n" +
+					"\n" +
+					"    var acr = selectAcrFrom(context, [\"acr1\", \"acr2\", \"acr3\",\"acr4\"]);\n" +
+					"\n" +
+					"    log.info(\"--------------- ACR selected: \"+acr);\n" +
+					"\n" +
+					"    context.setSelectedAcr(acr);\n" +
+					"\n" +
+					"    switch(acr) {\n" +
+					"        case \"acr1\" : executeStep({id :'1'});break;\n" +
+					"        case \"acr2\" : executeStep({id :'1'}); executeStep({id :'2'});  break;\n" +
+					"        case \"acr3\" : executeStep({id :'1'}); executeStep({id :'3'});  break;\n" +
+					"        case \"acr4\" : executeStep({id :'1'}); executeStep({id :'4'});  break;\n" +
+					"        default :  executeStep({id :'1'});  executeStep({id :'2'});  executeStep({id :'3'}); executeStep({id :'4'}); \n" +
+					"    }\n" +
+					"}\n",
+					"help": "imply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged",
+					"helpLink": "https://docs.wso2.com/display/IS540/WSO2+Identity+Server+Documentation"
```

Comment:
Would be better if we point this to the specific doc page, rather than pointing to the home page

Reply:
Help tip updated, Link can't be updated since the docs are still in WIP